osbs/conf.py

```diff
@@ -266,12 +267,22 @@ def get_registry_secrets(self):
         else:
             return []
 
-    def get_registry_api_versions(self):
+    def get_registry_api_versions(self, platform=None):
         value = self._get_value("registry_api_versions",
                                 self.conf_section,
                                 "registry_api_versions",
                                 default='v1,v2')
-        return value.split(',')
+        if not platform:
+            return value.split(',')
+        else:
+            self.get_platform_descriptors()
+            if platform in self.platform_descriptors and self.platform_descriptors[platform]['enable_v1']:  # noqa
+                return value.split(',')
```

Comment:
I don't think so. enable_v1 is "support for the Docker Registry HTTP API v1 may be included for this platform." v1 support is not mandatory.